src/language/providers/closeUtils.ts

```diff
@@ -336,7 +336,8 @@ export function getItemsForLineLT2(
             testLine = lineBefore
             while (!testText.includes('>')) {
               testText = document.lineAt(++testLine).text
-              if (testText.indexOf('<') > -1) [openTagArray.push(testLine)]
+              if (testText.indexOf('<' + nsPrefix + items[i]) > -1)
+                [openTagArray.push(testLine)]
```

Comment:
What is the reason for these square brackets here?!  I think it should just be `openTagArray.push(testLine)`, right?

Reply:
I removed the brackets.